jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorProcessor.java

```diff
@@ -110,7 +110,7 @@ public void process(Request request, Response response, Callback callback)
 
             try
             {
-                generateAcceptableResponse(request, response, code, message, cause, callback);
+                generateResponse(request, response, code, message, cause, callback);
```

Comment:
Unrelated to this PR.
Plus, this has "acceptable" in it's name as it's generating based on the `Accept` header.